mmv1/templates/terraform/pre_create/network_security_address_group_operation.go.erb

```diff
@@ -0,0 +1 @@
+project = ""
```

Comment:
Can we actually set this to `billingProject` for all 3 cases rather than `""`? It turns out there's currently an issue in every *other* resource and we should be sending that value through instead- we're waiting on operations with the wrong project override value if a `billing_project` is defined on the provider.

Reply:
It looks like [parent](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/networksecurity/AddressGroup.yaml#L65) in this resource should be set to project or organization. Is it safe to change the project to billingProject? 

I run [this test](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/networksecurity/resource_network_security_address_group_test.go#L12) locally, it seems like the test passes not matter what `project` is (empty string or billingProject).

Reply:
Yes! `billing_project` is specifically an unrelated project defined at the provider level used to charge quotas: https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#billing_project

This project field is being used to determine the user project override field to use- if a billing project is set, it should have that value. If one isn't, it should be empty (which it will be when one is not set)